staging/src/k8s.io/kms/encryption/cache.go

```diff
@@ -0,0 +1,39 @@
+package encryption
+
+import (
+	"encoding/base64"
+
+	"k8s.io/utils/lru"
+)
+
+type cache struct {
+	lruCache *lru.Cache
```

Comment:
Not a blocker for this but we're planning to move away from lru cache to use expiring cache in the kms v2 implementation (ref: https://github.com/kubernetes/kubernetes/issues/111920). We'll need to revisit this and check if we want to use expiring cache in reference implementation too.